app/controllers/admin/accounts_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 module Admin
   class AccountsController < BaseController
-    before_action :set_account, except: :index
+    before_action :set_account, except: %i(index new create)
```

Comment:
New and create don't have account IDs in the URL so this function isn't needed to be called